### DISCUSSION TITLE
Schwierigkeit bei Herausforderungen

### DISCUSSION THREAD

**Discussion by Goldkoloss** [Gorthian/Roll20_FSG_CharacterSheet#19] [Ideen]
Hallo, 
erstmal danke für die Arbeit die ihr euch mit dem Charsheet gemacht habt. Nun zu meinem Vorschlag:
Bei anderen Charakterbögen in Roll20 hab ich schon gesehen, dass beim klicken auf den Würfeln Button sich ein kleines Pop-up öffnet, in welchem die Schwierigkeit der Probe eingegeben werden kann. Nach dem Würfeln wird dann Automatisch bestimmt, wie Erfolgreich der Wurf war (In Erfolgsgrade ect., in diesem Fall dann in "Nein und", "Nein", "Nein, aber" ect.). 
Ich weiß nicht wie aufwendig das für euch wäre, aber vielleicht ist es ja eine Überlegung wert.
Mit freundlichen Grüßen,
Goldkoloss

**Comment by Gorthian** [comment]
Die Idee gefällt mir.
Ich werde mir die Tage mal anschauen ob und wie man das umsetzen kann.

**Comment by Gorthian** [comment]
Ist in die ToDo gewandert:
https://github.com/Gorthian/FSG_CharacterSheet_Roll20/issues/20

**Comment by Gorthian** [comment]
In meiner Entwicklerversion kann jetzt bei den Herausforderungen die Schwierigkeit ausgewählt werden und das entsprechende Ergebnis ("Nein, und", "Nein", usw) wird im Chat direkt mit ausgegeben.

Ich muss aber noch die genauen Schwellenwerte für Ergebnisse mit den Jungs von Donnerhaus abklären bevor ich es auch bei Roll20 einreichen werde.
